addins/Cake.Xamarin.Build.CakeBuilder.yml

```diff
@@ -0,0 +1,9 @@
+Name: Cake.Xamarin.Build.CakeBuilder
```

Comment:
Should be filtered as it is not an addin

Reply:
Done. Cake.Xamarin.CakeBuilder added to blacklist